Added the homepage

aka001 opened the pull request:
@noplay Kindly review the pr.

noplay commented on gns3server/handlers/__init__.py:
```diff
@@ -28,6 +28,20 @@
 from gns3server.handlers.api.config_handler import ConfigHandler
 from gns3server.handlers.api.server_handler import ServerHandler
 from gns3server.handlers.upload_handler import UploadHandler
+from ..config import Config
```
I think it's not require

noplay commented:
Thanks!

We can add pointers for usefull ressources:
* GN3 homepage http://gns3.com
* Support & Documentation: https://community.gns3.com/
* API documentation: http://api.gns3.net

noplay commented:
It's related to #99

aka001 commented:
@noplay  I have updated the pr and squashed the commits.
Do you want me to add the links for the "/upload" page on "home page", so that user can move there without anybody's help?

noplay commented:
Good idea for the /upload

aka001 commented:
@noplay I have updated the pr. Kindly review it.

noplay commented on gns3server/templates/homepage.html:
```diff
@@ -0,0 +1,18 @@
+    Welcome to Home page of GNS3.
+</h3>
+<h4>
+    Visit the upload page to try GNS3 features: <a href = "http://127.0.0.1:8000/upload">Upload</a>
```
You could put just /upload because the user can change the host and port.

aka001 replied:
@noplay True that. I have updated the pr and squashed the commits. Kindly review it.

aka001 commented:
@noplay Is there any problem with the pr? Shall I update it ?

noplay commented:
Sorry with the merge the PR has been autoclosed

noplay commented:
Can you make a new PR against master?

Thanks and sorry for the auto close

aka001 commented:
@noplay I am having my exams today. I will create the pr by tonight. Sorry for the delay.

aka001 commented:
@noplay I have created a new pull request. Kindly review it.
https://github.com/GNS3/gns3-server/pull/106
